### DISCUSSION TITLE
How do I install software on a Linux machine? Also, could you provide information on the feature plans for Linux?

### DISCUSSION THREAD

**Discussion by SAINI673** [leokhoa/laragon#740] [Ideas]
"How do I install software on a Linux machine? Also, could you provide information on the feature plans for Linux?"

**Comment by Pen-y-Fan** [comment]
Laragon is Windows only and can not be installed on Linux or Mac OS. There are no plans to create a Linux version.

Linux has package managers, to install PHP, Apache and MySQL. There are LAMP software packages too.

**Reply by SAINI673** [reply]
Ok Thanks

**Comment by rayio-dev** [comment]
A note for Linux users. I'm running Linux Mint and the Laragon portable version. Works great. I have been using it everyday, but I haven't tested everything. I'm using it mainly for mysql and I did add the mysql 5.7.44 version (Laragon portable only comes with 5.1, and had trouble loading version 8). Link and steps here for that:
https://gist.github.com/meorajrul/b57803bf1b4ddfd2f93e6ad37c3ac5f2
On Linux, you need Wine.

**Reply by SAINI673** [reply]
Can you send step how install in Linux machine 
Plz send screenshot also

Thanks

**Comment by ppfoong** [comment]
Which Linux distro are you referring to?

Most Linux distro will have the key components pre-installed, unless you selected not to install them during the installation process.

You can use the Linux distro's package manager to install/uninstall components.
